### PR TITLE
feat: alert-and-button

### PR DESCRIPTION
**Pull request by hrishi82**
Components Added:
- alert
- buttons

### CODE REVIEW

File: components/Buttons/buttons.css [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+}
+.outline-warning-btn:hover{
+    background-color: var(--warning-clr);
+    color: white;
+    border: none;
+}
+
+
+
+
```
Comment (romabulani) [inline components/Buttons/buttons.css:132]: Just a suggestion, dont give so much spacing between 2 properties

**Review by romabulani — APPROVED** [review]

File: components/Buttons/buttons.css (hunk 1 above)
Comment (SJTGSHIVAM) [reply components/Buttons/buttons.css:132]: I agree on that and the same goes with HTML too

**Review by SJTGSHIVAM — APPROVED** [review]
Things look good to me.

File: components/Buttons/buttons.css (hunk 1 above)
Comment (Logan1x) [reply components/Buttons/buttons.css:132]: I guess things like prettier remove these extra whitespace, maybe use them :)

**Review by Logan1x — APPROVED** [review]

File: components/Buttons/buttons.css (hunk 1 above)
Comment (hrishi82) [reply components/Buttons/buttons.css:132]: thank you for the suggestions. Will do!

**Pull request merged**